charts/seed-bootstrap/charts/cert-manager/templates/clusterissuer.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: ClusterIssuer
+metadata:
+  name: {{ .Values.clusterissuer.name }}
+spec:
+  acme:
+    email: {{ .Values.clusterissuer.acme.email }}
+    server: {{ .Values.clusterissuer.acme.server }}
+    privateKeySecretRef:
+      name: {{ .Values.clusterissuer.acme.privateKeySecretRef }}
+    dns01:
+      providers:
+      {{- range .Values.clusterissuer.acme.dns01.providers }}
+      - name: {{ .name }}
+      {{- if eq .type "aws-route53" }}
+        route53:
+          region: {{ .region }}
+          accessKeyID: {{ .accessKeyID }}
+          secretAccessKeySecretRef:
+            name: {{ .secretName }}
```

Comment:
As discussed: Let's create the secrets here as part of this Helm chart instead of doing it in the code.